src/server/api/organization.js

```diff
@@ -459,6 +459,25 @@ export const resolvers = {
         primaryColumn: "messaging_service_sid"
       });
     },
+    messagingServicesCount: async (organization, { active }, { user }) => {
+      const organizationId = parseInt(organization.id, 10);
+      try {
+        await accessRequired(user, organizationId, "ADMIN", true);
```

Comment:
required: Use UserRoleType enum.

Reply:
updated this and every other hard coded role string value in this file